test/e2e_node/memory_eviction_test.go

```diff
@@ -107,48 +84,39 @@ var _ = framework.KubeDescribe("MemoryEviction [Slow] [Serial] [Disruptive]", fu
 					return fmt.Errorf("expected 1 node, but see %d. List: %v", len(nodeList.Items), nodeList.Items)
 				}
 				node := nodeList.Items[0]
+
 				_, pressure := api.GetNodeCondition(&node.Status, api.NodeMemoryPressure)
-				if pressure != nil && pressure.Status == api.ConditionTrue {
-					return fmt.Errorf("node is still reporting memory pressure condition: %s", pressure)
+				if pressure == nil || pressure.Status != api.ConditionTrue {
+					return fmt.Errorf("node is not reporting memory pressure condition. Current Conditions: %+v", node.Status.Conditions)
 				}
 				return nil
-			}, 5*time.Minute, 15*time.Second).Should(BeNil())
+			}, 5*time.Minute, 5*time.Second).Should(BeNil())
+		})
+		It("should drop the node memory pressure condition", func() {
+			By("deleting the memory hog pod.")
+			graceperiod := int64(1)
+			err := f.PodClient().Delete(guaranteed.Name, &api.DeleteOptions{GracePeriodSeconds: &graceperiod})
```

Comment:
we will evict guaranteed pods.  this fails for me locally because the pod is evicted.  we agreed to evict guaranteed pods until we enforce memory limits on kubernetes daemons.